WcaOnRails/app/views/competitions/_competition_form.html.erb

```diff
@@ -223,19 +223,20 @@
         [
           t('.events'),
           true,
-          @competition.events.select(&:official?),
-          Event.all_official,
+          @competition.event_picker_events,
```

Comment:
Ah, `events_picker_events` returns an Array, so rather `.select(&:offical?)`, and the same below.